assets/src/domain/eventEditor/ui/tickets/ticketsList/actionsMenu/dropdown/TrashTicket.tsx

```diff
@@ -2,11 +2,12 @@ import React, { useCallback } from 'react';
 import { __ } from '@wordpress/i18n';
 
 import { Trash } from '@application/ui/layout/entityActionsMenu/entityMenuItems';
-import { useTicketContext } from '@edtrHooks/index';
 import { useTicketMutator } from '@edtrServices/apollo/mutations';
 
-const TrashTicket: React.FC = (props) => {
-	const { id } = useTicketContext();
+import { TicketMainMenuProps } from './types';
+
+const TrashTicket: React.FC<TicketMainMenuProps> = ({ ticket, ...props }) => {
+	const id = ticket?.id;
```

Comment:
Why optional chaining here? I think it's better to break the things here just on case instead of the `id` consumers

Reply:
https://github.com/eventespresso/event-espresso-core/pull/2727/commits/558e2a35883aff7fcaadd8957b623ac462f36c20

Reply:
done, thx